Lib/idlelib/textview.py

```diff
@@ -18,12 +20,9 @@ def __init__(self, parent, rawtext):
         super().__init__(parent)
         self['relief'] = 'sunken'
         self['height'] = 700
-        # TODO: get fg/bg from theme.
-        self.bg = '#ffffff'
-        self.fg = '#000000'
 
-        self.text = text = Text(self, wrap='word', highlightthickness=0,
-                                fg=self.fg, bg=self.bg)
+        self.text = text = Text(self, wrap='word', highlightthickness=0)
+        color_config(self.text)
```

Comment:
I think this should use `text` instead of `self.text`.  `self.text` was defined for testing purposes, but within this module, `text` is used to reference the widget.

Reply:
done